Lib/encodings/rot_13.py

```diff
@@ -47,7 +47,7 @@ def getregentry():
 
 ### Map
 
-rot13_map = codecs.make_identity_dict(range(256))
+rot13_map = codecs.make_identity_dict(list(range(256)))
```

Comment:
Why this change?  Was there a bug not mentioned?

Reply:
The function wants an iterable, there is no need to wrap range in list here like 2to3 would often unnecessarily do 🙂

Reply:
Hi,  I just made this change as 2to3 recommended,   no bug here.

Reply:
I’m saying 2to3 is over-cautious, but human judgment shows that this change is not needed here.

```suggestion
rot13_map = codecs.make_identity_dict(range(256))
```